docs/content/latest/reference/configuration/yb-tserver.md

```diff
@@ -674,6 +674,35 @@ Default: `256MB`
 
 ---
 
+### Network compression
+
+Use the following two gflags to configure RPC compression:
+
+##### --enable_stream_compression
+
+Controls whether YugabyteDB uses RPC compression. If you set this flag to `true`, you must also specify a stream compression algorithm.
+
+##### --stream_compression_algo
+
+Specifies which compression algorithm to use, if stream compression is enabled. Valid values are:
+
+- 0: no compression
+- 1: gzip
+- 2: snappy
+- 3: lz4
+
+In most cases, LZ4 (`--stream_compression_algo=3`) offers the best compromise of compression performance versus CPU overhead.
+
+{{< note title="Upgrade notes" >}}
+
+To upgrade from an older version that doesn't support RPC compression (such as 2.4), to a newer version that does (such as 2.6), you need to do the following:
+
+1. Rolling restart to upgrade YugabyteDB, such as from 2.4 to 2.6.
+1. Rolling restart to enable compression, on both master and tserver, by setting `enable_stream_compression=true`.
```

Comment:
We should mark somehow that this step is optional, if the version you're upgrading to already has this enabled by default, eg: if you're upgrading to the next stable release of 2.6 >= 2.6.3, or any 2.8 version, you can skip this step